doc-content/enterprise-only/bpmn/bpmn-lanes-ref.adoc

```diff
@@ -1,2 +1,9 @@
-= Swimlanes 
-Swimlanes are process elements that visually group tasks related to one group or user. You can use user tasks in combination with swimlanes to assign multiple human tasks to the same actor. At runtime, lanes auto-claim or assign tasks to users who have completed another task in that lane within the same process instance. When the first task in a swimlane is created, and that task has an actor Id specified, that actor Id is  assigned to all other tasks of that swimlane as well.
\ No newline at end of file
+= BPMN2 swimlanes in process designer 
+
+Swimlanes are process elements that visually group tasks related to one group or user. You can use user tasks in combination with swimlanes to assign multiple human tasks to the same actor. At runtime, swimlanes auto-claim or assign tasks to users who have completed another task in that lane, within the same process instance. When the first task in a swimlane is created, and that task has an actor Id specified, that actor Id is  assigned to all other tasks of that swimlane as well. A lane is a sub-partition within a process that enables you to group some process elements and define their common parameters.
```

Comment:
Search "ID" and "Id" throughout. Should be "ID" unless referencing the UI or something, and therefore would likely be monospace. Here should be "actor ID", I'm guessing.